pkg/restic/config.go

```diff
@@ -64,10 +64,14 @@ func getRepoPrefix(location *velerov1api.BackupStorageLocation) (string, error)
 	switch backendType {
 	case AWSBackend:
 		var url string
+		var region string
 		switch {
 		// non-AWS, S3-compatible object store
 		case location.Spec.Config["s3Url"] != "":
 			url = location.Spec.Config["s3Url"]
+		case location.Spec.Config["region"] != "":
+			region = location.Spec.Config["region"]
+			url = fmt.Sprintf("s3-%s.amazonaws.com", region)
```

Comment:
This can be simplified to keep all the region inferring/getting logic together by moving this into the `default` case.
```
default:
    region := location.Spec.Config["region"]
    var err error
    if region == "" {
        region, err = getASBucketRegion(bucket)
        if err != nil {
		url = "s3.amazonaws.com"
		break
	}
	url = fmt.Sprintf("s3-%s.amazonaws.com", region)
    }
```

Reply:
Let's get rid of the switch statement altogether and just use an if/else clause.  I don't think we want any additional cases here anyhow.